tests/script-based-pre/kani_list/list.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# Check that the JSON file produced by `kani list` is correct.
+# Note that the list.expected file omits the value for "kani-version"
+# to avoid having to update the test every time we bump versions.
```

Comment:
They're actually slightly different scripts -- one invokes `kani` and other invokes `cargo kani`. We have separate tests for each because `cargo kani` creates multiple `KaniMetadata` objects, and we want to test that we correctly combine the information from multiple metadata objects.